package.json

```diff
@@ -24,22 +28,42 @@
     "redis",
     "Google Cloud Memorystore for Redis API"
   ],
+  "contributors": [
+    "Chris Wilcox <crwilcox@google.com>"
+  ],
+  "scripts": {
+    "test": "repo-tools test run --cmd npm -- run cover",
+    "generate-scaffolding": "repo-tools generate all",
+    "generate-scaffolding-samples": "repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json",
+    "cover": "nyc --reporter=lcov mocha --require intelli-espower-loader test/*.js && nyc report",
+    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
+    "test-no-cover": "repo-tools test run --cmd mocha -- test/*.js --no-timeouts",
+    "lint": "repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/ smoke-test/",
+    "prettier": "repo-tools exec -- prettier --write src/*.js src/*/*.js samples/*.js samples/*/*.js test/*.js test/*/*.js system-test/*.js system-test/*/*.js smoke-test/*.js",
+    "docs": "repo-tools exec -- jsdoc -c .jsdoc.js",
+    "system-test": "repo-tools test run --cmd mocha -- system-test/*.js smoke-test/*.js --no-timeouts"
+  },
   "dependencies": {
     "google-gax": "^0.14.2",
```

Comment:
That's actually a bug in the generator template. Please use the latest `"^0.16.1" otherwise it will use an ancient gRPC version.

Reply:
Just submitted https://github.com/googleapis/gapic-generator/pull/1976 to fix it.